docs/my-website/release_notes/v1.79.1-stable/index.md

```diff
@@ -54,6 +54,62 @@ pip install litellm==1.80.0
 
 ---
 
+### Performance – 19× Lower Median Latency, 13× Lower p95 Latency
+
+This update fixes two major issues. First, embeddings now go through the same optimized pipeline as chat/completions, benefiting from all performance improvements applied so far. Second, while the responses API was already using the same pipeline, it now also includes our internal network management for connection handling, drastically improving latency and reducing memory overhead from per-request setup.
+
+As a result, performance improved across all latency percentiles:
+
+`/embeddings`
```

Comment:
it's not clear to me what is the litellm overhead here and it's not clear what is being measured, is this e2e latency or overhead ?